### ISSUE TITLE
Attempt to index local 'pos' (a nil value) on neocodeium/lua/neocodeium/doc.lua:78

### ISSUE THREAD

**Issue by eofferma** [monkoose/neocodeium#31]
### 👎 Problem

Hi!,

Great plugin thanks!

I'm experiencing an error when I updated to v1.4.0. The error was the following:

```
Error  Error executing vim.schedule lua callback: ....local/share/nvim/lazy/neocodeium/lua/neocodeium/doc.lua:78: attempt to index local 'pos' (a nil value)
stack traceback:
	....local/share/nvim/lazy/neocodeium/lua/neocodeium/doc.lua:78: in function 'get'
	.../share/nvim/lazy/neocodeium/lua/neocodeium/completer.lua:243: in function 'request'
	.../share/nvim/lazy/neocodeium/lua/neocodeium/completer.lua:305: in function <.../share/nvim/lazy/neocodeium/lua/neocodeium/completer.lua:304>
Error  Se ha detectado un error al procesar User Auto-órdenes para "_NeoCodeiumCompleterStatus":
```

When I downgrade to v1.3.0 the problem goes away.

### 🔢 Steps to reproduce

1. Install NeoCodeium v1.4.0
2. Wait for NeoCodeium to suggest a completion
3. Accept the completion

### 👍 Expected behavior

That the error does not occur

### Neovim version (nvim -v)

NVIM v0.10.1

### Operating system (uname -om)

Linux x86_64

### :checkhealth neocodeium

OK

### Relevant log output

```shell
NeoCodeium: log file is empty
```

**Comment by LeonardoMor** [comment]
Seeing the exact same behavior.

**Comment by tufei** [comment]
Exact same issue on Fedora 39/40

**Comment by monkoose** [comment]
Sorry for inconvenience. Should be fixed with the last commit c768da5773e9aea550ec6eab0a4a50351a2de8d6
